frontend/src/components/Table.tsx

```diff
@@ -78,7 +84,7 @@ const Table = <T,>({
   return (
     <>
       {(toolbarContent || showPagination) && (
-        <Toolbar>
+        <Toolbar customChipGroupContent={<></>}>
```

Comment:
Remove the clear filters -- may want to configure the table to disable it? There is a need for the clear filters to have functionality here -- but it is well removed from the owner. Some design thought may be needed here.

cc @Gkrumbach07 let me know if you want to be part of that conversation / brainstorming.